src/chrome/content/rules/Geekzone.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="geekzone.co.nz (partial cdn)">
+	<target host="cdn.geekzone.co.nz" />
+	<target host="images.geekzone.co.nz" />

```

Comment:
Thanks. Yes, I can list live.geekzone.co.nz and testip.geekzone.co.nz as 301 for the whole sub-domain. But what about domains with only certain pages (login) redirect to https? Shall I list these URLs on target list or list in the comment only?